src/StoryWrightProcessor/PlayWrightExecutor.ts

```
@@ -30,76 +30,136 @@ export class PlayWrightExecutor {
   }
 
   mouseDownAsync = async (selector: string) => {
-    let element;
-    if (selector.charAt(0) === '#') {
-      element = await this.page.$(`id=${selector.substring(1, selector.length)}`)
-    } else {
-      element = await this.page.$(`${selector}`);
-    }
-    const box = await element.boundingBox();
-    await this.page.mouse.move(box.x + box.width / 2, box.y + box.height / 2);
-    await this.page.mouse.down();
+    try{  
+      let element;
+      if (selector.charAt(0) === '#') {
+        element = await this.page.$(`id=${selector.substring(1, selector.length)}`)
+      } else {
+        element = await this.page.$(`${selector}`);
+      }
+      const box = await element.boundingBox();
+      await this.page.mouse.move(box.x + box.width / 2, box.y + box.height / 2);
+      await this.page.mouse.down();
+    }catch (err){
+      console.error("ERROR: mouseDown: ", err.message);
```

We should also throw the error again from here. And in our main processor log the error about which story failed. That can be helpful in reporting how many stories succeded vs failed.

---

Added **'throw err'** to all catch blocks